jobserver/templates/job_request_detail.html

```diff
@@ -38,6 +38,21 @@ <h2 class="mb-4">
       <code><small class="text-muted">{{ jobrequest.identifier }}</small></code>
     </h2>
 
+    <p>
+      This page shows the technical details of what happened when authorised researcher {{ jobrequest.created_by.username }} requested one or
```

Comment:
You could use the `name` property (instead of `username`) on User here, it gets the Users full name falling back to their username if they don't have that set.

Reply:
Oh that's great. Thanks. Done.